platform/ios/docs/pod-README.md

```diff
@@ -33,7 +33,17 @@ bash "${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/Mapbox.framework/strip-fra
 
 If your application targets iOS 7.x, you’ll need to install the static framework instead:
 
-1. Open the project editor and select your application target. Drag `Mapbox.framework` from the `static` folder into the “Embedded Binaries” section of the General tab. (Don’t drag it into the “Linked Frameworks and Libraries” section; Xcode will add it there automatically.) In the sheet that appears, make sure “Copy items if needed” is checked, then click Finish.
+1. Build from source manually per above.
```

Comment:
Which step “above” are we referring to? Unless we’re going to stop distributing a zipped static build (which would be fine by me), these instructions should point the user to the GitHub releases page and the prebuilt static build, not tell them how to build from source.